package/README.adoc

```diff
@@ -0,0 +1,140 @@
+shares, although other non-cryptographic unique files may also become necessary over time.
+
+In order to satisfy the above requirements of building and deploying a complex distributed system
+consisting of unique, private files, two CLI tools have been created:
+
+  . link:src/bin/omicron-package.rs[omicron-package] - build, package, install on local machine
+  . link:src/bin/thing-flinger.rs[thing-flinger] - build, package, deploy to remote machines
+
+
+If a user is working on their local illumos based machine, and only wants to run
+omicron in single node mode, they should follow the install instruction in
+the link:../README.adoc[Omicron README] and use `omicron-package`. If the user
+wishes for a more complete workflow, where they can code on their local laptop,
+use a remote build machine, and install to multiple machines for a more realistic
+deployment, they should use `thing-flinger`.
+
+The remainder of this document will describe a typical workflow for using
+thing-flinger, pointing out room for improvement.
+
+== Environment and Configuration
+
+
+     +------------------+                +------------------+
+     |                  |                |                  |
+     |                  |                |                  |
+     |      Client      |---------------->     Builder      |
+     |                  |                |                  |
+     |                  |                |                  |
+     +------------------+                +------------------+
+                                                   |
+                                                   |
+                                                   |
+                                                   |
+                       +---------------------------+--------------------------+
+                       |                           |                          |
+                       |                           |                          |
+                       |                           |                          |
+              +--------v---------+       +---------v--------+       +---------v--------+
+              |                  |       |                  |       |                  |
+              |                  |       |                  |       |                  |
+              | Deployed Server  |       | Deployed Server  |       | Deployed Server  |
+              |                  |       |                  |       |                  |
+              |                  |       |                  |       |                  |
+              +------------------+       +------------------+       +------------------+
+
+
+`thing-flinger` defines three types of nodes:
+ * Client - Where a user typically edits their code and runs thing-flinger. This can run any OS.
+ * Builder - A Helios box where Omicron is built and packaged
+ * Deployment - Helios machines where Omicron will be installed and run
```

Comment:
Might suggest calling this `Deployed server` to match the above diagram.

Reply:
fixed